flybywiresim/a32nx
# refactor: (ecam) Convert FCTL page to React

Fixes #[issue_no]

## Summary of Changes

Refactor FCTL page to use Reach and new SimVars. Changes should be invisible to user.

## Screenshots (if necessary)



![ECAM-FCTL-engine2](https://user-images.githubusercontent.com/5834254/112836535-a56de600-9092-11eb-9839-bdf62d832adb.png)
![ECAM-fctl-off](https://user-images.githubusercontent.com/5834254/112836536-a6067c80-9092-11eb-96d7-a621f18c88af.png)
![ECAM-FCTL-right](https://user-images.githubusercontent.com/5834254/112836539-a6067c80-9092-11eb-9667-c379f0827af9.png)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Start engines
2. Check all control service movements (ailerons, elevators, rudder and spoilers) are shown in the ECAM
3. Check spoilers show when armed speedbrake deploys


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
